components/home/Challenge.tsx

```diff
@@ -14,6 +14,7 @@ export function Challenge({ children, title }: PropsWithChildren & { title: stri
     const router = useRouter();
 
     const [isOpen, setIsOpen] = useState(false);
+    const[isLiked, setIsLiked] = useState(false);
```

Comment:
We will have to think about how to store the likes. Will we do it by post? Every like is stored in the post? Or by user? Every like the user has done in one place. I think the latter might be a better idea

Reply:
I am working on it now ! On my branch you'll see you can be on the Home Screen as a current logged in user, and I will do the same for the challenges.